src/buildStyles.mjs

```diff
@@ -0,0 +1,87 @@
+import { join, basename, dirname } from 'path';
+import { writeFileSync, mkdirSync } from 'fs';
+import autoprefixer from 'autoprefixer';
+import postcss from 'postcss';
+import notifier from 'node-notifier';
+import glob from 'glob';
+import sass from 'sass';
+
+/**
+ * Use a simple script to build styles as the CLI version of SASS does e.g.
+ * - not support globs
+ * - not support a central configuration of the output directory (instead the out path must be
+ * specified for every file)
+ * - not watch included files (chould be done through e.g. chokidar)
+ */
+const buildStyles = async({
+    destinationFolder = '.',
+    sourceFolder = '.',
+    sourceFiles = [],
+    compress = false,
+    showNotifications = false,
+} = {}) => {
+
+    // De-glob files
+    const files = sourceFiles
+        .map((file) => join(sourceFolder, file))
+        .map((fileWithPath) => glob.sync(fileWithPath))
+        .flat();
+
+    // Process files
+    const parsedCSSPromises = files.map(async(file) => {
+
+        const cssFileName = basename(file).replace(/\.scss$/, '.css');
+        const destinationFilePath = join(destinationFolder, cssFileName);
+
+        const sassOptions = { sourceMap: true, style: (compress ? 'compressed' : 'expanded') };
+        const sassResult = sass.compile(file, sassOptions);
```

Comment:
Werden für SASS auch Fehler und Warnings in der Konsole ausgegeben?

Reply:
Jap: Wenn ich hier auf L2 einen invaliden String habe, failed der Build:

<img width="882" alt="image" src="https://user-images.githubusercontent.com/1211144/204996972-41ae548d-6301-4bd7-ba97-e622b525fb5d.png">